pk-test/docker.lua

```diff
@@ -0,0 +1,191 @@
+--------------------------------------------------------------------------------
+-- docker.lua
+-- This file is a part of pk-test library
+-- Copyright (c) pk-test authors (see file COPYRIGHT for the license)
+-- See file `COPYRIGHT` for the license
+--------------------------------------------------------------------------------
+
+require 'lua-aplicado.module' -- for import()
+require 'lfs' -- for lfs.chdir(), lfs.currentdir()
```

Comment:
Guidelines dictate that you must cache the module locally. (Please make sure that the guidelines indeed say so explicitly, send a PR to fix if they do not.)

`local lfs = require 'lfs'`